src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/upstart/start-template

```diff
@@ -27,7 +27,11 @@ end script
 # set the working directory of the job processes
 chdir ${{chdir}}
 
+setuid ${{daemon_user}}
```

Comment:
You don't want to do `sudo chown ${{daemon_user}}:${{daemon_group}} -R /usr/share/${{app_name}}l/` 
`/usr/share/<app>` is generally owned by `root`. If you need write access to disk you add `linuxPackageMappings` with exactly the permissions you need.